### PR TITLE
Fix add Selected button, to abjust working with wysiwyg and widgets

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1068: The Image from Enhanced Media Gallery is not added via WYSIWYG

### Manual testing scenarios (*)

1. Installed Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Enable Enhanced Media Gallery in Stores -> Configuration -> Advanced -> System
4. From Admin go to Content - Pages, click Add New Page
5. Expand Content
6. Click Insert/edit Image
7.Click Search source
**The Image
DeepinScreenshot_select-area_20200331131050
 is added and a link appears in the Source field.**

**Scenario 2**
1. From Admin go to Content - Pages, click Add New Page
2. Expand Content
3. Click Show/Hide editor 
4.Click Insert Image 
**Proper img tag generated, and image is displayed after switch to the wysiwyg**

DeepinScreenshot_select-area_20200331130732
DeepinScreenshot_select-area_20200331130821

### CODE REVIEW

File: MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/insertImageAction.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        },
+
+        /**
+         * Return opener Window object if it exists, not closed and editor is active
+         *
+         * @param {String} targetElementId
+         * return {Object|null}
+         */
+        getMediaBrowserOpener: function (targetElementId) {
+            if (typeof wysiwyg != 'undefined' &&
```
Comment (VladimirZaets) [inline MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/insertImageAction.js]: Can you please explain why we check is Wysiwyg and tinyMceEditors exists? If I correct to understand, wysiwyg can be except in the case when it is disabled in the configuration, correct? in this case, we should pass is wysiwyg enabled option by the configuration, not just check is JS class exists. It's not very important for now, but we need at least should create task for that @sivaschenko

File: MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/insertImageAction.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        /**
+         * Get target element
+         *
+         * @param {String} targetElementId
+         * @returns {*|n.fn.init|jQuery|HTMLElement}
+         */
+        getTargetElement: function (targetElementId) {
+            var opener;
+
+            if (typeof wysiwyg != 'undefined' && wysiwyg.get(targetElementId)) {
```
Comment (VladimirZaets) [inline MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/insertImageAction.js]: The same situation here.

File: MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js [hunk 3]
```diff
@@ -7,17 +7,19 @@ define([
...
+], function ($, _, Element, $t, addSelected) {
     'use strict';
 
     return Element.extend({
         defaults: {
             modalSelector: '',
             template: 'Magento_MediaGalleryUi/image/actions',
-            targetElementId: null,
             mediaGalleryImageDetailsName: 'mediaGalleryImageDetails',
+            imageModelName: 'media_gallery_listing.media_gallery_listing.media_gallery_columns.thumbnail_url',
```
Comment (VladimirZaets) [inline MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js]: We should move it to XML configuration

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @Nazar65. Please take a look at my comments below

File: MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js (hunk 3 above)
Comment (Nazar65) [reply MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js]: ok

File: MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/insertImageAction.js (hunk 1 above)
Comment (Nazar65) [reply MediaGalleryUi/view/adminhtml/web/js/grid/columns/image/insertImageAction.js]: thank you, i agree with you, for now i added a proper check with isUndefined, and later i might refactor it to use configuration settings.

**Pull request merged**